README.md

```diff
@@ -1,14 +1,17 @@
-README
+# ProActive Workflows & Scheduling
```

Comment:
I was confirmed  "ProActive Scheduling & Orchestration" is only the name of one of our partals, part of the "ProActive Workflows & Scheduling" solution.